CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ This project adheres to [Semantic Versioning], with the exception that minor rel
 - ♻️ Update the `measure` operations in the MLIR dialects to no longer support more than one qubit being measured at once ([#1106]) ([**@DRovara**])
 - ✨ Add MQT's implementation of a QDMI Device for neutral atom-based quantum computing ([#996]) ([**@ystade**])
 - ✨ Add `qubit` operation to the `MQTRef` (previously `MQTDyn`) dialect for static qubit addressing ([#1098]) ([**@MatthiasReumann**])
+- ✨ Add `qubit` operation to the `MQTOpt` dialect for static qubit addressing ([#1116]) ([**@MatthiasReumann**])
```

Comment:
As stated in the comment at the top of the file, new contributions should go to the top of this list. Could you please fix that?

Reply:
done.